domain/src/main/java/com/distillery/android/domain/FakeToDoRepository.kt

```diff
@@ -0,0 +1,91 @@
+package com.distillery.android.domain
+
+import com.distillery.android.domain.models.ToDoModel
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.receiveAsFlow
+import kotlinx.coroutines.launch
+import java.util.Date
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.atomic.AtomicLong
+
+private val uniqueIdGenerator = AtomicLong(0)
```

Comment:
I think ids are unique by nature so we can avoid the prefix
```suggestion
private val idGenerator = AtomicLong(0)
```
Also, what's the reason for not moving it inside the class? Should we comment that in the code?

Reply:
@0neel as it is a fake - I decided that we should have somewhat guaranteed unique ids, and static counter is a good way to achieve that.